portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetsFilters.testcase

```diff
@@ -0,0 +1,214 @@
+			AssertTextEquals.assertText(
+				locator1 = "Message#EMPTY_STATE_INFO",
+				value1 = "No default filters were created.");
+
+			AssertTextEquals.assertText(
+				locator1 = "Message#EMPTY_STATE_TEXT",
+				value1 = "Start creating a Filter to display specific Data.");
+		}
+	}
+
+	@description = "LPS-183476. Confirm that the New filter modal is closed and the changes are not applied when canceling on Cancel button"
+	@priority = 3
+	test CanCancelFilterCreationOnCancelButton {
+		task ("And clicks to add a new filter") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("Then click on the "Cancel button"") {
+			Button.clickCancel();
+		}
+
+		task ("And assert the New filter modal is closed and the changes are not applied") {
+			AssertElementNotPresent(
+				key_modalTitle = "New Filter",
+				locator1 = "Button#CLOSE_MODAL");
+
+			AssertTextEquals.assertText(
+				locator1 = "Message#EMPTY_STATE_INFO",
+				value1 = "No default filters were created.");
+		}
+	}
+
+	@description = "LPS-183476. Confirm that the New filter modal is closed and the changes are not applied when canceling on Close (X) button"
+	@priority = 3
+	test CanCancelFilterCreationOnCloseButton {
+		task ("And clicks to add a new filter") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("Then click on the "Close button"") {
+			Click(
+				key_modalTitle = "New Filter",
+				locator1 = "Button#CLOSE_MODAL");
+		}
+
+		task ("And assert the New filter modal is closed and the changes are not applied") {
+			AssertElementNotPresent(
+				key_modalTitle = "New Filter",
+				locator1 = "Button#CLOSE_MODAL");
+
+			AssertTextEquals.assertText(
+				locator1 = "Message#EMPTY_STATE_INFO",
+				value1 = "No default filters were created.");
+		}
+	}
+
+	@description = "LPS-183476. Confirm that the text 'Select' is labeling the  'Filter By Field'"
+	@priority = 5
+	test CanCreateFilters {
+		task ("And adds a new filter") {
+			FrontendDataSetAdmin.createSingleFilter(
+				filterName = "Test Filter",
+				key_filterType = "id");
+		}
+
+		task ("Then asserts the selected option is actually displayed") {
+			AssertElementPresent(
+				chosenFilter = "id",
+				key_tableColumn = 1,
+				key_tableRow = 1,
+				locator1 = "FrontendDataSetAdmin#FILTER_TABLE_CELL");
+		}
+	}
+
+	@description = "LPS-183476. Confirm that 'Filter By' field cannot be empty"
+	@priority = 3
+	test CannotLeaveFilterByEmpty {
+		task ("And adds a new filter") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("And Fill the name filter with Test filter") {
+			Type(
+				locator1 = "TextInput#NAME",
+				value1 = "Test Filter");
+		}
+
+		task ("Then Comfirm that is not possible clicks save button") {
+			AssertElementPresent(
+				key_text = "Save",
+				locator1 = "Button#ANY_DISABLED");
+		}
+	}
+
+	@description = "LPS-183476. Confirm that is possible search filter when click on search icon"
```

Comment:
Put the correct LPS--> LPS-176184 (in Poshi, we should put the LPS of the original story/bug)